src/SpectatorView.Native/README.md

```diff
@@ -33,6 +33,26 @@ If you are using an Elgato capture card, you will need to clone Elgato's [gameca
 - Update the Elgato_Filter user macro in `dependencies.props` with the corresponding path on your computer.
 - Restart Visual Studio
 
+**Azure Kinect**
+If you are using an Azure Kinect, you will need to download or clone the Azure Kinect Sensor SDK with [Microsoft installer](https://download.microsoft.com/download/e/6/6/e66482b2-b6c1-4e34-bfee-95294163fc40/Azure%20Kinect%20SDK%201.3.0.exe) or [github repo](https://github.com/microsoft/Azure-Kinect-Sensor-SDK)
```

Comment:
>github [](start = 245, length = 6)

I know the rest of this page has this wrong too, but this should be GitHub. Maybe we can make sure the other instances are fixed too.